addons/website/static/src/snippets/s_instagram_page/options.js

```diff
@@ -62,11 +57,11 @@ options.registry.InstagramPage = options.Class.extend({
         this.$target[0].dataset.instagramPage = widgetValue || "";
         // As the public widget restart is disabled for instagram, we have to
         // manually restart the widget.
-        await this.trigger_up("widgets_start_request", {
+        this.website.websiteRootInstance.trigger_up("widgets_start_request", {
```

Comment:
Why was the await removed?

Reply:
Because trigger_up never returns a Promise

Reply:
🤦🏻 good catch. I think we should await the `widgets_start_request` like we do [here](https://github.com/odoo/odoo/blob/ea39f5f8e9e86ad8138acaf8e95f057e7d97e3cd/addons/website/static/src/js/editor/snippets.options.js#L830) for example. And this should probably be fixed in stable too 👍🏻

Reply:
I've pushed an update, should be good!

Reply:
Awesome thanks! Did you also do the fix in stable? It's probably not that big of a deal since it doesn't cause a bug, but it's still wrong and is a quick win IMO.

Reply:
Oh I can do that! Thanks for telling me :) I'll try to think about this kind of thing from now on.
https://github.com/odoo/odoo/pull/171166